.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Thank you for opening a Pull Request!
```

Comment:
General comment:  We want users to start pull requests from a recent point of the project.  We have seen users not pick up from after we have done some cleanup.  We end up having to remerge changes on our end.  I'm not sure how to best nudge contributors in this direction as it is likely that they will have a wide range of skill levels with git and github.

Reply:
I'd like to add a section to the README on how to contribute, see the note [here](https://github.com/google/earthengine-community/pull/348#discussion_r644349790)

We could also add some check boxes to the PR template

[ ] Before editing your code, did you ensure your fork is current
[ ] Do the files all have copywrite and license header
[ ] etc

Reply:
The presence of the headers should be auto-checked by some kind of presubmit. I think Gino is working on it?

Reply:
Points ack and I agree.

RE need to sync fork to head before branch, edit, PR: I'm writing a contributing.md file that will state this. We can maybe link to it in the PR template or have a checklist in the PR, but at this point (PR opened) sort of too late.

RE presence of the headers should be auto-checked: yes, @gino-m is looking into it. I'm adding this as a section of the contributing.md file - every files needs a license header. Reviewers should initialize a review with a checklist - Gino has one for community tutorials, but we've not been using it consistently: go/ee-community/tutorials#triaging-pull-requests